Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -346,11 +346,10 @@ static void launchWithExtensionKit(ProcessLauncher& processLauncher, ProcessLaun
         xpc_dictionary_set_bool(bootstrapMessage.get(), "disable-logging", disableLogging);
     }
 
-    bool isWebKitDevelopmentBuild = ![[[[NSBundle bundleForClass:NSClassFromString(@"WKWebView")] bundlePath] stringByDeletingLastPathComponent] hasPrefix:FileSystem::systemDirectoryPath()];
-    if (isWebKitDevelopmentBuild) {
-        xpc_dictionary_set_fd(bootstrapMessage.get(), "stdout", STDOUT_FILENO);
-        xpc_dictionary_set_fd(bootstrapMessage.get(), "stderr", STDERR_FILENO);
-    }
+#if !defined(WEBKIT_IS_PRODUCTION_BUILD)
+    xpc_dictionary_set_fd(bootstrapMessage.get(), "stdout", STDOUT_FILENO);
+    xpc_dictionary_set_fd(bootstrapMessage.get(), "stderr", STDERR_FILENO);
+#endif
```

Comment:
This is a behavior change when, for instance, an Apple engineer downloads a production root and converts it into a relocatable bundle. Also, STP and downlevel roots are "production" but not installed to /System. So I don't think you can assume that production implies "system WebKit".

Reply:
Yeah I am aware of that. I think the question is more whether we want a production root to be telling the auxiliary process to forward stdout/stderr to UIProcess's stdout/stderr or not.

I could pretty easily just keep this logic the same without the WEBKIT_IS_PRODUCTION_BUILD thing, let me take a look into it.